hiredis_redismodule.h

```diff
@@ -0,0 +1,97 @@
+#ifndef __HIREDIS_REDISMODULE_H__
```

Comment:
This should become a standard hiredis adapter at some point.

Reply:
You mean merging into hiredis? Maybe.